webapp/src/index.tsx

```diff
@@ -28,6 +28,7 @@ import {
 } from './shared/urls';
 import { AuthProvider } from './shared/context/auth-context';
 import RequireAuth from './shared/components/RequireAuth/RequireAuth';
+import ChatRoom from './pages/ChatRoom/ChatRoom';
```

Comment:
in line 16, better, I think. We should add it to pages

Reply:
Thanks! Done.